src/Storefront/Theme/Command/ThemeDumpCommand.php

```diff
@@ -97,4 +97,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         return 0;
     }
+
+    private function getTechnicalName(string $themeId): ?string
+    {
+        $technicalName = null;
+
+        do {
+            /** @var ThemeEntity|null $theme */
+            $theme = $this->themeRepository->search(new Criteria([$themeId]), $this->context)->first();
+            $technicalName = $theme->getTechnicalName();
```

Comment:
You should first check if `$theme` is actually an instance of `ThemeEntity`. You already check for `!is_null($theme)` in the loop condition, but that won't be enough, since you call `getTechnicalName` on `$theme` here.

Reply:
You are right. Had this in my mind, forgot to implement though 🙈

Reply:
Fixed.